CodeReview/src/CodeReviewManager.h

```diff
@@ -28,19 +28,41 @@
 
 #include "codereview_api.h"
 
+#include "VersionControlUI/src/nodes/DiffFrame.h"
+
 #include "nodes/CommentedNode.h"
 
 namespace CodeReview {
 
+using DiffFrames = QList<VersionControlUI::DiffFrame*>;
+
+using OrderedDiffFrames = QList<VersionControlUI::DiffFrame*>;
```

Comment:
Yes, that sounds good.

Functions should definitely have names defined with `using`, but for the other types here, that seems unnecessary.